tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +91,494 @@ mvn package
+```properties
+spring.couchbase.bootstrap-hosts=DB_CONN_STR
+spring.couchbase.bucket.user=DB_USERNAME
+spring.couchbase.bucket.password=DB_PASSWORD
+```
 
 ## Running The Application
 
+### Directly on the machine
+
 At this point the application is ready, and you can run it via your IDE or from the terminal:
 
 ```shell
 mvn spring-boot:run -e -X
 ```
 
-> Note: Couchbase Server 7 must be installed and running on localhost (http://127.0.0.1:8091) prior to running the Spring Boot app.
+<!-- Screeenshot -->
+
+![Spring Boot Application](./app-startup-spring-boot.png)
+
+> Note: Either the Couchbase Server must be installed and running on localhost or the connection string must be updated in the `application.properties` file.
 
 Once the site is up and running you can launch your browser and go to the Swagger Start Page]: `http://localhost:8080/swagger-ui/` to test the APIs.
 
+<!-- Screenshot -->
+
+![Swagger UI](./swagger-documentation-spring-boot.png)
+
+### Docker
+
+You can also run the application using Docker. The `Dockerfile` is provided in the root directory of the project. To build and run the Docker image, use the following commands:
+
+```shell
+docker build -t springboot-couchbase .
+```
+
+```shell
+docker run -p 8080:8080 springboot-couchbase
+```
+
 ## What We'll Cover
 
-A simple REST API using Spring Boot and the Couchbase SDK version 3.x with the following endpoints:
+A simple REST API using Spring Boot and the `Couchbase SDK version 3.x` with the following endpoints:
 
-- [POST a Profile](#post-a-profile) – Create a new user profile
-- [GET a Profile by Key](#get-a-profile-by-key) – Get a specific profile
-- [PUT Profile](#put-profile) – Update a profile
-- [DELETE Profile](#delete-profile) – Delete a profile
-- [GET Profiles by Searching](#get-profiles-by-searching)  – Get all profiles matching first or last Name
+- [Create new airlines with essential information](#post-mapping).
+- [Update airline details](#put-mapping).
+- [Delete airlines](#delete-mapping).
+- [Retrieve airlines by ID](#get-mapping).
+- [List all airlines with pagination](#get-mapping).
+- [List airlines by country](#custom-sql-queries).
+- [List airlines by destination airport](#custom-sql-queries).
 
-## Document Structure
+### Document Structure
 
-We will be setting up a REST API to manage some profile documents. Our profile document will have an auto-generated UUID for its key, first and last name of the user, an email, and hashed password. For this demo we will store all profile information in just one document in a collection named `profile`:
+We will be setting up a REST API to manage some airline documents. Our airline document will have a structure similar to the following:
 
 ```json
 {
-  "pid": "b181551f-071a-4539-96a5-8a3fe8717faf",
-  "firstName": "John",
-  "lastName": "Doe",
-  "email": "john.doe@couchbase.com",
-  "password": "$2a$10$tZ23pbQ1sCX4BknkDIN6NekNo1p/Xo.Vfsttm.USwWYbLAAspeWsC"
+  "id": "airline_8091",
+  "type": "airline",
```

Comment:
type is a legacy field that we have skipped in these quick starts. It should not be required for creating new docs.

Reply:
removed type